utils/hoursUtils.test.js

```diff
@@ -0,0 +1,131 @@
+jest.mock('airtable')
+const hoursUtils = require('./hoursUtils')
+
+const testHours = require('../testData/hours_response.json')
+
+
+const expectedHours = {
+    hours: [
+        {
+            day: 'sunday',
+            dayDigit: 0,
+            openTime: undefined,
+            closeTime: undefined,
+            isToday: false,
+            hoursSummary: 'Closed'
```

Comment:
`day` and `hoursSummary` feel like something that should be taken care of on the UI side, not the backend, for two reasons:
* The text would likely need to be localized for languages
* Time is formatted differently for different languages

The UI should be able to recreate the same information based on the `dayDigit`, `openTime`, `closeTime` (and the subsequent structure for multiple time periods).

Reply:
I thought about that too but... in our case, 0 always means sunday, 1 always means monday...on the UI, when we go to display the word "Monday" that can be translated to other languages, but on the backend, the monday hours will still always be connected to day_digit 1.

The data model here is largely based off of Google Places API opening_hours field: https://developers.google.com/maps/documentation/places/web-service/details?hl=vi and that api does accept a language parameter but that looks like it only affects that order of events in the array, but the hours all still come back xAM - xPM... do you know what kinds of differences might exist for formatting different languages?

It's true that it doesn't strictly need to be here. If it's in the frontend though, the logic of figuring out Open 24 hours or Closed would also have to be there too though, right?

Reply:
As far as formatting, not all cultures us `AM` and `PM` as distinctions.  Even in the US, the military uses 24hr time (e.g. Open from 0800 - 1800).

For the logic bits, It depends on what's on the Airtable side, but I would model them as boolean values (e.g. `is24hrs`, `isPermClosed`).

Reply:
Sure, in that case, should probably use Luxon on the frontend too then to do that formatting. 

Airtable is right :) What does it depend on from Airtable? boolean values sounds fine, will jus need to parse them on the frontend then.

The `but in our case` was in reference to `The text would likely need to be localized for languages` was that talking about the `day` value? I'm confused what part of day would be localized for language.

I think part of why I wanted to do hoursSummary here is so that the frontend doesn't have to deal with multitime periods but I suppose we can just send an array of intervals in stead of an array of strings back when we get there.

Reply:
Yeah, removed `day` from the formatted hours, realized I wasn't even using it actually. 
Thanks for those examples about the AM/PM - that's exactly what i was looking for!

Yepp, the rest of that all makes sense!

Removed the `summary` field. For now, the goal is to allow the automation to happen without changing how anything is represented on the frontend, but when we start actually displaying a weekly schedule, then will plan on using Luxon there and likely adding language compatibility (as well as timezone flexibility 🤞 )